datalabeling/beta/cloud-client/src/main/java/com/example/datalabeling/CreateAnnotationSpecSet.java

```diff
@@ -39,29 +40,38 @@ static void createAnnotationSpecSet(String projectId) {
     annotationLabels.put("label_1", "label_1_description");
     annotationLabels.put("label_2", "label_2_description");
 
-    try (DataLabelingServiceClient dataLabelingServiceClient = DataLabelingServiceClient.create()) {
+    String endpoint = System.getenv("DATALABELING_ENDPOINT");
+    if (endpoint == null) {
+      endpoint = DataLabelingServiceSettings.getDefaultEndpoint();
+    }
 
+    try (DataLabelingServiceClient dataLabelingServiceClient =
```

Comment:
Please add comments to each file, that this is unnecessary for users to do or something to that effect.

Reply:
+1. We should also add a comment why this is necessary for these snippets, and try to hide this portion from being staged in the documentation.

Reply:
nit: It would be nice to still add a comment here (doesn't need to be in the staged version) to explain the extra code.